CONTRIBUTING.md

```diff
@@ -11,7 +11,7 @@ There are only a few guidelines that we need contributors to follow.
 Below are the details about how to set up the JDT Language Server in different IDEs. **Notice**: You only need set up it once in any of your preferred IDE.
 
 ### A) Setting up the JDT Language Server in VS Code
-1) Please install [Eclipse PDE support](https://marketplace.visualstudio.com/items?itemName=yaozheng.vscode-pde) extension in your VS Code first. The PDE extension's home page provides more usage details about _Reload Target Platform_, _Run JUnit Plug-in Test_, _Run Eclipse Application_.
+1) Please install [Eclipse PDE support][1] extension in your VS Code first. The PDE extension's home page provides more usage details about _Reload Target Platform_, _Run JUnit Plug-in Test_, _Run Eclipse Application_.
```

Comment:
Nice. So the Markdown format knows to use links referenced in a "footnote". Good to know.